### PR TITLE
make hill pass through the origin

### PR DESCRIPTION
**Pull request by juanitorduz**
Closes https://github.com/pymc-labs/pymc-marketing/issues/917 by adding a constant

----
📚 Documentation preview 📚: https://pymc-marketing--920.org.readthedocs.build/en/920/

### CODE REVIEW

File: pymc_marketing/mmm/transformers.py [hunk 1]
```diff
@@ -908,7 +908,7 @@ def hill_saturation(
     r"""Hill Saturation Function
 
     .. math::
-        f(x) = \frac{\sigma}{1 + e^{-\beta(x - \lambda)}}
+        f(x) = \frac{\sigma}{1 + e^{-\beta(x - \lambda)}} - \frac{\sigma}{1 + e^{\beta\lambda}}
```
Comment (wd60622) [inline pymc_marketing/mmm/transformers.py:911]: How does this effect the parameter interpretations that are listed in the docstring below?

**Review by wd60622 — COMMENTED** [review]
Should the SaturationTransformation priors change with this change?

File: pymc_marketing/mmm/transformers.py (hunk 1 above)
Comment (juanitorduz) [reply pymc_marketing/mmm/transformers.py:911]: Ah! I did not look into the docstrings 🙈. I will work on them. In essence we are adding an offset. Should not change that much. 

I do think this has to be corrected as these saturation functions should pass through the origin.

Comment (wd60622) [reply pymc_marketing/mmm/transformers.py:911]: Totally!

**Comment by wd60622** [comment]
The plot of the current priors as reference: https://pymc-marketing--920.org.readthedocs.build/en/920/api/generated/pymc_marketing.mmm.components.saturation.HillSaturation.html

File: pymc_marketing/mmm/transformers.py (hunk 1 above)
Comment (juanitorduz) [reply pymc_marketing/mmm/transformers.py:911]: Done!

**Comment by juanitorduz** [comment]
> The plot of the current priors as reference: https://pymc-marketing--920.org.readthedocs.build/en/920/api/generated/pymc_marketing.mmm.components.saturation.HillSaturation.html

Looks good as they all pass through the origin as expected. I constrained the priors a bit.

**Review by wd60622 — APPROVED** [review]

**Pull request merged**